### PR TITLE
Test against python 3.9

### PR DESCRIPTION
**Pull request by clbarnes**
A very minor update to be pushed through the CI sausage machine.

### CODE REVIEW

**Comment by florisla** [comment]
Very nice, thanks! I put this on my list to review soon(*).

(*) Might not be really soon.

**Comment by clbarnes** [comment]
This will probably be superceded by #229 .

**Comment by clbarnes** [comment]
I reverted the 3.9 addition to that PR, and so will open a fresh one once that's merged.

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ commits and tags:
 * works without any VCS, but happily reads tag information from and writes
   commits and tags to Git and Mercurial if available
 * just handles text files, so it's not specific to any programming language
-* supports Python 3 and PyPy3
+* supports non-EOL Python (>=3.6 at time of writing) and PyPy3
```
Comment (florisla) [inline README.md:25]: Any particular reason to stop testing 3.5?  It's EOL at Python.org but some long-term-support distros might still support it.

Of course, enabling f-strings counts as a good reason ;-)

**Review by florisla — CHANGES_REQUESTED** [review]
Needs a rebase now that we use Github Actions.

**Pull request closed without merging**